src/java/org/apache/cassandra/net/InboundConnectionInitiator.java

```diff
@@ -445,11 +454,11 @@ void setupMessagingPipeline(InetAddressAndPort from, int useMessagingVersion, in
             InboundMessageHandler handler =
                 settings.handlers.apply(from).createHandler(frameDecoder, initiate.type, pipeline.channel(), useMessagingVersion);
 
-            logger.info("{} connection established, version = {}, framing = {}, encryption = {}",
+            logger.info("{} messaging connection established, version = {}, framing = {}, encryption = {}",
```

Comment:
before

```
127.0.0.2:7000(127.0.0.1:65115)->127.0.0.1:7000-LARGE_MESSAGES-f53d678a connection established, version = 12, framing = CRC, encryption = enabled (openssl)
```

after

```
127.0.0.2:7000(127.0.0.1:65115)->127.0.0.1:7000-LARGE_MESSAGES-f53d678a messaging connection established, version = 12, framing = CRC, encryption = enabled (openssl)
```

Reply:
personal preference, I prefer the old

Reply:
It's to disambiguate between streaming connections so I'd prefer to keep it.